app/javascript/packs/admin.js

```diff
@@ -1,8 +1,50 @@
-import 'utils/request';
-import 'admin/dashboard/index';
-import 'admin/stats/index';
-import 'admin/dashboard/unattached_promo_registration'
-import Rails from 'rails-ujs';
+import "utils/request";
+import "admin/dashboard/index";
+import "admin/stats/index";
+import "admin/dashboard/unattached_promo_registration";
+import Rails from "rails-ujs";
+// import Tribute from "tributejs";
+// import "tributejs/dist/tribute.css";
+
+import Avatar from "../views/admin/components/userNavbar/components/TopNav/Avatar.svg";
+
+// document.addEventListener("DOMContentLoaded", function() {
+//   if (window.location.href.indexOf("admin/publishers/") !== -1) {
```

Comment:
Perhaps delete?

Reply:
sorry this is while i was fixing the builds 😅